### DISCUSSION TITLE
COM Port access denied

### DISCUSSION THREAD

**Discussion by Apowell219** [arkypita/LaserGRBL#2045] [General]
I just finished extending the Y-axis to my Atomstack M50 laser.  Now when I go into laserGrbl to connect it, I get an error message telling me access to COM port 3 is denied.  I also tried connecting to COM port 4 but also received the same message.  How do I overcome this?

**Comment by arkypita** [comment]
Not a lasergrbl issue.
Close any other program that keep the COM port open (or reboot PC if you don't know what program is keep port open)
